libcxx/include/__atomic/atomic_init.h

```diff
@@ -15,13 +15,13 @@
 #  pragma GCC system_header
 #endif
 
-#define ATOMIC_FLAG_INIT {false}
-#define ATOMIC_VAR_INIT(__v) {__v}
+#define ATOMIC_FLAG_INIT                                                                                               \
+  { false }
+#define ATOMIC_VAR_INIT(__v)                                                                                           \
+  { __v }
 
 #if _LIBCPP_STD_VER >= 20 && !defined(_LIBCPP_DISABLE_DEPRECATION_WARNINGS)
-# if defined(_LIBCPP_CLANG_VER) && _LIBCPP_CLANG_VER >= 1400
 #  pragma clang deprecated(ATOMIC_VAR_INIT)
-# endif
 #endif // _LIBCPP_STD_VER >= 20 && !defined(_LIBCPP_DISABLE_DEPRECATION_WARNINGS)
```

Comment:
As it's only one line inside `#if` now, we can remove the comment after `#endif`. It doesn't make it easier to rad.